content/blog/2020/07/2020-07-30-winsw-yaml-support-2.adoc

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: "Windows Service Wrapper : YAML Configuration Support - GSoC Phase - 02 Updates"
+tags:
+- winsw
+- windows
+- jenkins
+- gsoc
+- gsoc2020
+author: buddhikac96
+description: "This blog post discusses about phase 02 updates of Windows Service Wrapper Yaml support project of GSoC - 2020"
+opengraph:
+  image: /images/post-images/2020-07-08-winsw-yaml-support/opengraph.png
+---
+
+Hello, world! GSoC 2020 Phase 2 has ended now and it was a great period for WinSW - YAML COnfigurations support project. 
+In this blog post, I will announce the updates during the GSoC 2020 - Phase 2. If you are not already aware of this project, 
+I would recommend reading https://www.jenkins.io/blog/2020/07/08/winsw-yaml-support/[this blog post] which was published after GSoC 2020 - Phase 1.
+
+== Project Scope
+
+* Windows Service Wrapper - YAML configuration support
+* New CLI
```

Comment:
Would be reat to expand a bit to explain what;s new

Reply:
Since it is included in my previous blog post I did not add a long description about new CLI. How ever I update it into more decriptive one that before. would that be enough or would you prefer description with details about commands and options ?